ocaml/idl/datamodel.ml

```diff
@@ -5984,7 +5984,8 @@ let pool_enable_ha = call
   ~in_oss_since:None
   ~params:[
     Set(Ref _sr), "heartbeat_srs", "Set of SRs to use for storage heartbeating.";
-    Map(String, String), "configuration", "Detailed HA configuration to apply"]
+    Map(String, String), "configuration", "Detailed HA configuration to apply";
+    String, "cluster_stack", "HA Cluster manager stack";]
```

Comment:
The idea was that we could do this a bit more dynamically. The cluster-stack param gets used as a path to a set of scripts that define the ha interface - so you can add a new stack by dumping a load of scripts to a particular place in the filesystem without having to change xapi.